domains/xgeneric.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "xgeneric",
+           "discord": "1194494419064328296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.D232zLsD29QIoCDQBNjD76jn7VgIPpSOIrRPBMn37VV_JTS_3fJAKK7ADQtFwQu0A-I0TcZTU3UczrEGdwjmkHHAhPu95BuRbFqbWruAnX6ETw0JjSJQvYmlrFjF1xDMM33CJ3GW0R985HyTcB5MbU6-QvWbqPou5bwRShlQ06KKP944EiASahDDNAog2HUwEf7VJjWxdB-hP3Yj_BOvlC_kQBfZBhOTG4OWwbvbAonKjK26oqrVdUKhuDT0VfvI4VsBXlSG3g8P2LnyZZPmh3xJ3UQ4aWYRFhJkC3f0M98NQ4mI7ddwZu4YPVcw9yHiMD9iDnTBnNiSsJVZeMhqQw.6v7QP1iKQEXF2xIliVLoxA.n1gPGIhivHdAaq9enhjewqRKzW2BtPsOt97Ng4x7yp8nv-OzAS4Jev1hp_M3z0XL7l5z1Wc2YrPW7z3trdUlmTZkCLcOziKhwwECaKbQNErOq3yXDxe7K5FvhmQWGNQJ.fZiP_f8z_xpVs5jS0MRQPQ"
+        },
+    
+        "record": {
+            "CNAME": "paulswebhelp.com"
```

Comment:
Unable to access site. Provide a preview.

Reply:
Here is a preview: 
![image](https://github.com/user-attachments/assets/a37cc75f-1f69-44ff-8186-ac36b8c45e1f)